csrankings-o.csv

```diff
@@ -167,11 +168,13 @@ Onur Tolga Sehitoglu,Middle East Technical University,http://user.ceng.metu.edu.
 Onyeka Ezenwoye,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=oezenwoye,WzzzVvoAAAAJ
 Ophir Frieder,Georgetown University,http://people.cs.georgetown.edu/~ophir,uUXCSMkAAAAJ
 Or D. Dantsker,Indiana University,https://luddy.indiana.edu/contact/profile/index.html?Or_Dantsker,gohaofoAAAAJ
+Or Litany,Technion,https://orlitany.github.io/,Ihs8dwsAAAAJ

```

Comment:
Please verify >= 75% time appointment at Technion ("I'm a Senior Research Scientist at [NVIDIA](https://nv-tlabs.github.io/) and an Assistant Professor at the [Technion](https://www.cs.technion.ac.il/people/faculty/#:~:text=of%20visual%20tasks.-,Or%20Litany,-Senior%20Lecturer),")

Reply:
Or replies: "I’m 80% technion" (this means he spends one day a week at NVIDIA)